### PR TITLE
#167510760 Implement route to get articles of authors a user follows

### PR DESCRIPTION
**Pull request by successgilli**
#### What does this PR do?
This pull request creates a route to fetch all articles belonging to authors a registered user is following
#### Description of Task to be completed
- create route to fetch articles followed by user
- add pagination to articles feed
#### How should this be manually tested?
- clone the branch
- run 'npm i', 'npm run db:migrate' and 'npm run start:dev'
- open the route GET http://localhost:3000/api/v1/articles/feed on postman
#### Any Background Context You want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[#167510760](https://pivotaltracker.com/story/show/167510760)

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/controllers/article.js [hunk 1]
```diff
@@ -367,5 +369,35 @@ export default {
...
+    */
+  getUserFeed: async (request, response) => {
+    const { id: userId } = request.user;
+    const { page = 1, limit = 10 } = request.query;
+    const followedAuthors = await Followers.findAllFollowees(userId);
+    const followedAuthorsId = followedAuthors.map(eachAuthor => eachAuthor.followeeId);
+    const { data } = await paginator(Articles, {
+      where: {
+        authorId: followedAuthorsId,
+        status: 'draft'
```
Comment (successgilli) [inline src/controllers/article.js]: change to published latter

File: src/models/Followers.js [hunk 2]
```diff
@@ -57,4 +57,18 @@ export default class Followers extends Model {
       onDelete: 'CASCADE'
     });
   }
+
+  /** Returns article array
+ *
+ * @function
+ *
+ * @param {Array} id - id of articles to be sort
```
Comment (successgilli) [inline src/models/Followers.js]: change type to string and the description should be changed to match the expected value.
Id should be changed to userId.

File: src/models/Followers.js [hunk 3]
```diff
@@ -57,4 +57,18 @@ export default class Followers extends Model {
       onDelete: 'CASCADE'
     });
   }
+
+  /** Returns an array of authors the user is following
```
Comment (kennyedward) [inline src/models/Followers.js:62]: Good job Gilbert. However, can you please align the JSDoc comment to meet with the standard set in the team convention documentation. Thanks

**Review by kennyedward — APPROVED** [review]
Please review the comment I left on the code.

File: src/models/Followers.js (hunk 3 above)
Comment (successgilli) [reply src/models/Followers.js:62]: I see that , 

> Good job Gilbert. However, can you please align the JSDoc comment to meet with the standard set in the team convention documentation. Thanks

I see that @kennyedward, I wonder why my Eslint didn't flag it.
Thanks

**Review by Merkll — APPROVED** [review]
Good job

**Pull request merged**